source/acknowledgements-python.md

```diff
@@ -0,0 +1,25 @@
+---
+jupytext:
+  cell_metadata_filter: -all
+  formats: py:percent,md:myst,ipynb
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.13.8
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Acknowledgments for the Python Edition
+
+We'd like to thank everyone that has contributed to the development of 
+[*Data Science: A First Introduction (Python Edition)*](https://ubc-dsci.github.io/introduction-to-datascience-python/).
+This is an open source Python translation of the original [*Data Science: A First Introduction*](https://datasciencebook.ca);
+the original focused on the R programming language. Both of these books are 
+used to teach DSCI 100, a new introductory data science course 
+at the University of British Columbia (UBC).
+
+We will finalize this acknowledgements section after the book is complete!
```

Comment:
It might be nice to include an initial acknowledgment to Gloria and Nadia? But we can also do this later when we have had a proper discussion about this

Reply:
I agree totally that we should put Gloria and Navya up front in the acks -- but yeah let's punt that for later.

Reply:
ah on second thought let me add something first, and then we can refine later